docs/index.html

```diff
@@ -56,7 +56,7 @@ <h2>Get started now</h2>
       <div class="steps">
         <div class="step step-1">
           <!--label>Step 1</label-->
-          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=v0.2.0&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.2.0" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
+          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=v0.2.0&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.2.0&cloudshell_tutorial=../docs/tutorial.md" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
```

Comment:
You'll have to update this link and others to use the new release

Reply:
Okay! This PR is still in-scope.

I updated the code. In order to test it, try using this URL: https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=walkthrough-tutorial&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.2.5&cloudshell_tutorial=tutorial.md

Reply:
I'm a bit confused with the new website link. Don't we need the website in the `/docs` folder for it to be picked up by GitHub poages? Why do we have two different index.htmls now?

Reply:
Sorry for not responding sooner! I got caught up in telemetry work last week.

You're right, it should be in the `docs/` folder. I made the directory changes in a PR merged this afternoon. The directory is to keep the AppEngine website separate from the documentation of the User Guide, for ease-of-use and clarity. It was the result of a request on a previous PR that Simon made--to make the website into its own directory. Unfortunately, in that previous PR the URL was not ready to point at the AppEngine deployment yet, so it did not get merged then.